formidable/views.py

```diff
@@ -239,19 +240,31 @@ def build(self, role):
     def get_formidable_object(self, kwargs):
         return Formidable.objects.get(pk=kwargs['pk'])
 
-    def get(self, request, **kwargs):
+    def post(self, request, **kwargs):
         try:
             formidable = self.get_formidable_object(kwargs)
         except Formidable.DoesNotExist:
             raise exceptions.NotFound()
 
         form_class = self.get_form_class(formidable)
-        form = self.get_form(form_class)
+        form = self.get_form(form_class, request)
         if form.is_valid():
             return self.form_valid(form)
         else:
             return self.form_invalid(form)
 
+    def get(self, request, **kwargs):
+        """
+        GET method is deprecated in favor of POST
+
+        """
+        warnings.simplefilter('always', category=DeprecationWarning)
```

Comment:
why adding a simple filter here? let the user's default, it should be fine.

Reply:
It was done this way on PeopleAsk, I removed it